webapp/channels/src/components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -350,7 +349,7 @@ class AdvancedCreateComment extends React.PureComponent<Props, State> {
     };
 
     saveDraftOnUnmount = () => {
-        if (!this.isDraftEdited || !this.state.draft) {
+        if (!this.isDraftEdited || !this.state.draft || this.props.rootDeleted) {
```

Comment:
Added the check here to not update draft on unmount

Reply:
If I understand this right, it's to ensure that we don't remove the draft, close the RHS, and then immediately have this component re-save the draft. Is that right?

Reply:
Yes, thats right.